tests/datalab/test_datalab.py

```diff
@@ -809,14 +809,25 @@ def test_incremental_search(self, pred_probs, random_embeddings):
         lab = Datalab(data=data, label_name="labels")
         lab.find_issues(features=random_embeddings)
         summary = lab.get_issue_summary()
-        assert len(summary) == 3
-        assert "label" not in summary["issue_type"].values
+        assert len(summary) == 4
+        assert "label" in summary["issue_type"].values
         lab.find_issues(pred_probs=pred_probs, issue_types={"label": {}})
         summary = lab.get_issue_summary()
         assert len(summary) == 4
         assert "label" in summary["issue_type"].values
-        label_summary = lab.get_issue_summary("label")
-        assert label_summary["num_issues"].values[0] > 0
+        label_summary_pred_probs = lab.get_issue_summary("label")
+        assert label_summary_pred_probs["num_issues"].values[0] > 0
+        lab.find_issues(
+            features=random_embeddings, pred_probs=pred_probs, issue_types={"label": {}}
+        )
+        summary = lab.get_issue_summary()
+        assert len(summary) == 4
+        assert "label" in summary["issue_type"].values
+        label_summary_both = lab.get_issue_summary("label")
+        assert (
+            label_summary_both["num_issues"].values[0]
+            == label_summary_pred_probs["num_issues"].values[0]
+        )
```

Comment:
[nit] I'm not entirely sure if these tests fall under `test_incremental_search`, I think checking that executing `find_issues` with pred_probs and executing `find_issues` with both returns the same result falls under a different test